NEWS.md

```diff
@@ -1,4 +1,23 @@
-# cchsflow 1.0.1 (Latest build)
+# cchsflow 1.1.0 (Latest build)
+2020-01-08
+
+## Features
+- Updated `DESCRIPTION` to expand description of package and added The Ottawa
+Hospital as author
+- Changed `data_source` parameter in rec_with_table() to `data`; and changed
+order of parameters to have `variables` after `dataset_name`
+- Included automated testing for functions
+- Refactored functions to account for age below 0 and include warning messages
```

Comment:
name of the exact function that was modified

Reply:
Suggest that we group items into clearer categories, in order from most important to least important.

## Breaking changes

## New features

## Minor improvements and bug fixes